src/scalsource.f90

```diff
@@ -2,7 +2,23 @@
 !!tg3315, 8 Apr 2016 
```

Comment:
Author: Tom Grylls to be consistent with other files.

Other files I have authored (e.g. modchem) do not have the author name. For consistency perhaps add my name to that. Or also I am equally happy for my author names to be removed across all files as many files have had secondary input across the uDALES team, which isn't captured.

Reply:
Happy either way -- these files/headers would require a lot of clean up. In this PR I was just trying to address the licencing and copyright issue but I can remove the username and just leave the uDALES Team bit if you are happy with that. What about the other authors/contributors? Shall we leave them as they are for now and address that in a future PR?

Reply:
Yes that is fine by me - let's remove the username here now for consistency. Authors/ contributors can be adjusted if needed in a future PR.

Reply:
There are a lot of instances in a lot of files as comments -- do I need to remove those too or shall I just remove the ones in the header?

Reply:
No I just meant in this header

Reply:
Thanks!

Reply:
👍 Cool. Fixed in 4725cf6. BTW re generating paper add `from branch joss-paper` to the usual command.

Reply:
Thanks!!!